### PR TITLE
Added support for retrieving operational data via gRPC/YANG

### PR DESCRIPTION
**Pull request by chburket**
Also added show_running_yang utility to get XR YANG configuration from bash shell.

Currently running rake test, but initial tests look fine.

### CODE REVIEW

File: docs/README-utilities.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Executable utilities included in this gem
+
+## check_metric_limits.rb
```
Comment (chburket) [inline docs/README-utilities.md:3]: Just to be clear, I didn't create the check_metric_limits.rb utility. I just found it in the bin directory and figured I would document it along with our new show_running_yang utility.

File: lib/cisco_node_utils/node.rb [hunk 2]
```diff
@@ -260,7 +265,7 @@ def product_id
           # No support for structured output for this command yet
           output = get(command:     'show inventory',
                        data_format: :cli)
-          return /NAME: "Rack 0".*\nPID: (\S+)/.match(output)[1]
+          return /NAME: .*\nPID: (\S+)/.match(output)[1]
```
Comment (chburket) [inline lib/cisco_node_utils/node.rb:268]: Made the regex a little less specific, since we had a case where the output did not include the "Rack 0" part.

**Comment by chburket** [comment]
Rake test results are consistent with previous runs.

**Comment by mikewiebe** [comment]
👍 @chburket I did not conduct a deep dive on this one but mostly was looking for impacts to Nexus.  Based on your test results etc the changes look fine to me.

**Pull request merged**